src/javascripts/example.js

```diff
@@ -0,0 +1,16 @@
+import * as utils from './utils.js'
+
+function ExamplePage ($module) {
+  this.$module = $module
+}
+ExamplePage.prototype.init = function () {
+  var $module = this.$module
+  if (!$module) {
+    return
+  }
+  var $form = $module.querySelector('form[action="/form-handler"]')
+  utils.preventFormSubmission($form)
```

Comment:
Is there a reason this is split out into a separate function and file? It's not re-used anywhere?

Reply:
fine, I'll fold it into example.js